sql/executor_test.go

```diff
@@ -113,9 +113,13 @@ func TestExecutorTrainAndPredictDNN(t *testing.T) {
 	a.NotPanics(func() {
 		stream := runExtendedSQL(testTrainSelectIris, testDB, modelDir, nil)
 		a.True(goodStream(stream.ReadAll()))
-
 		stream = runExtendedSQL(testPredictSelectIris, testDB, modelDir, nil)
 		a.True(goodStream(stream.ReadAll()))
+
+		stream = runExtendedSQL(testClusteringTrain, testDB, modelDir, nil)
```

Comment:
Maybe we need to put this case into another test case, it's not DNN.

Reply:
Done.